config/locales/defaults/en.yml

```diff
@@ -102,6 +102,20 @@ en:
         Lots of words on a page can make content difficult for users to read online. The longer the
         page is, the less usable it is likely to be. You could investigate whether this content could
         be shorter. Otherwise, does it have headings and bulletpoints to help users scan?
+    reading_time:
+      title: 'Reading time'
+      short_title: 'Reading time'
+      summary: 'Typical time to read all %{number_of_words} words'
+      context: ''
+      unit: ''
+      data_source: none
+      external_link: ''
+      about_title: 'About reading time'
+      about: >
+        Reading time is calculated by dividing the word count for the page by an average reading speed
+        of 200 words per minute. The time it takes to read the page is rounded up to the nearest minute.
+        This calculation doesn’t take into account how complicated the content is or any of the page’s
+        attachments.
```

Comment:
@pollygreen have you had a chance to look at this copy?

Reply:
@pollygreen wrote it!